### PR TITLE
feat(sdk-coin-sol): create method to produce broadcastabel sol sweep txn

### PR DESCRIPTION
**Pull request by alia-bitgo**
TICKET: [WP-488](https://bitgoinc.atlassian.net/browse/WP-488?atlOrigin=eyJpIjoiYTYzMWVhN2EzMmFlNGE5Mjk3MTRlYmU3ODgyODU3ZTEiLCJwIjoiaiJ9)

[WP-488]: https://bitgoinc.atlassian.net/browse/WP-488?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/sdk-coin-sol/src/sol.ts [hunk 1]
```diff
@@ -152,6 +153,38 @@ interface RecoveryOptions {
...
+}
+
+interface TxRequest {
+  transactions: OvcTransaction[];
+  walletCoin: string;
+}
+
+interface OvcTransaction {
+  unsignedTx: SolTx;
+  signatureShares: any[];
```
Comment (margueriteblair) [inline modules/sdk-coin-sol/src/sol.ts:173]: Is there any way to make this a type that's not `any`?

File: modules/sdk-coin-sol/test/unit/sol.ts [hunk 2]
```diff
@@ -1509,6 +1509,80 @@ describe('SOL:', function () {
       sandBox.restore();
     });
 
+    it('should take OVC output and generate a signed sweep transaction', async function () {
```
Comment (margueriteblair) [inline modules/sdk-coin-sol/test/unit/sol.ts:1512]: nit: tests for non-happy paths

Comment (alia-bitgo) [reply modules/sdk-coin-sol/test/unit/sol.ts:1512]: yeah we have tested e2e with sol, and it works

File: modules/sdk-core/src/index.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { EddsaUtils } from './bitgo/utils/tss/eddsa/eddsa';
 export { EddsaUtils };
 import { EcdsaUtils } from './bitgo/utils/tss/ecdsa/ecdsa';
 export { EcdsaUtils };
-export { GShare, SignShare, YShare } from './account-lib/mpc/tss/eddsa/types';
+export { GShare, SignShare, YShare, Signature as edDSASignature } from './account-lib/mpc/tss/eddsa/types';
```
Comment (zahin-mohammad) [inline modules/sdk-core/src/index.ts:12]: Types should be Pascal cased (first letter should be capitalized).

```suggestion
export { GShare, SignShare, YShare, Signature as EddsaSignature } from './account-lib/mpc/tss/eddsa/types';
```

or 
```suggestion
export { GShare, SignShare, YShare, Signature as EDDSASignature } from './account-lib/mpc/tss/eddsa/types';
```

**Review by zahin-mohammad — CHANGES_REQUESTED** [review]

**Review by rlutzbitgo — APPROVED** [review]

**Review by zahin-mohammad — APPROVED** [review]

**Pull request merged**